.eslintrc.json

```diff
@@ -106,7 +106,9 @@
     "react/jsx-filename-extension": ["error", { "extensions": [".jsx"] }],
     "react/jsx-tag-spacing": "error",
     "react/jsx-wrap-multilines": "error",
-    "react/self-closing-comp": "error"
+		"react/self-closing-comp": "error",
+		"react/no-unused-prop-types": ["error", { "skipShapeProps": false }]
+
```

Comment:
This shows errors if no prototypes are declared?

Reply:
That rule is already in, this one makes sure you don't forget some props you declared or defined in types but not used in your code.